configuration/filesystems.md

```diff
@@ -0,0 +1,53 @@
+# Gravwell Indexer Supported Filesystems
+
+Gravwell Indexers require robust, seekable, and POSIX complaint filesystems in order to function properly.  The Gravwell system makes extensive use of memory mapping, madvise calls, and filesystem specific optimizations to maximize compression ratios and query throughput.  Picking a good filesystem for your deployment is critical to ensuring a manageable and fast Gravwell system.
+
+Gravwell officially supports the following Linux filesystems.
+
+| Filesystem | Minimum Kernel Version | Supports Transparent Compression |
+|:-----------|:-----------------------|:--------------------------------:|
+| EXT4       | 3.2                    |                                  |
+| XFS        | 3.2                    |                                  |
+| BTRFS      | 5.0                    | ✅                               |
+| ZFS        | N/A                    | ✅                               |
+| NFSv4      | N/A                    |                                  |
+
+
+
+
+### Ext4
+
+The Ext4 filesystem is well supported and an excellent default choice as a backing filesystem.  Ext4 supports volume sizes up to 1EiB and up to 4 Billion files, Gravwell extensively tests on Ext4.
+
+### XFS
+
+The XFS filesystem is extremely fast, well tested, and praised by kernel developers.  XFS supports a wide array of configuration options to optimize the filesystem for specific storage device topology.
+
+### BTRFS
+
+The BTRFS filesystem has been a core part of the Linux kernel for over a decade, but due to some rocky starts and conservative warnings about stability early on in its life cycle it gets a bad rap.  Gravwell extensively tests the BTRFS filesystem in transparent compression topology and has found it to be exceedingly fast, memory efficient, and well supported.  While BTRFS is supported all the way back to Linux Kernel 3.2, 5.X and newer kernels contain a highly optimized and stable code base.  Gravwell recommends BTRFS with ZSTD compression for a hot store when transparent compression is enabled and users want the best performance.
```

Comment:
Similar here:
Maybe "tests the BTRFS filesystem in ***a*** transparent compression topology" ?
Or maybe "tests the BTRFS filesystem ***with a*** transparent compression topology" ?